bin/launch_interpret_json.py

```diff
@@ -42,23 +40,29 @@ def interpret_json(input_json: dict) -> list:
     # combine split possibilities with noise ones in a all vs all manner, each splitter wil be assigned to each noiser
     list_of_json_to_write = []
 
-    # Check if both lists are empty
-    if not list_noise_combinations and not list_split_combinations:
-        list_of_json_to_write.append({"experiment": schema.experiment})
+    # The  pipeline has always to happen at least once, aka on the data itself untouched. this sould always be passed through the pipeline. This line is creating the json for that.
+    list_of_json_to_write.append({"experiment": schema.experiment, "noise": None, "split": None})
```

Comment:
then we are gonna have at least one json with no noise and split?
Then the pipeline will run on the original data without split as a must?